app/models/login_user.rb

```diff
@@ -7,7 +7,7 @@ def perform
   private
 
   def user
-    @_user ||= User.find_or_create_by(github_id: auth_hash[:uid])
+    @_user ||= FindOrRegisterUser.new(github_id: auth_hash[:uid]).perform
```

Comment:
we are still doing a `create` at https://github.com/18F/micropurchase/blob/0f1958ca89b8778516b6b105f904cd9ddf5b6f77/app/models/user.rb#L14

let's move that oauth logic into `FindOrRegisterUser`